### PR TITLE
adding ACs to the spec

### LINKED ISSUE #826: 0011-MARA-check_order_allocate_margin needs acceptance criteria
In order to get QA test coverage [this spec](https://github.com/vegaprotocol/specs-internal/blob/master/protocol/0011-MARA-check_order_allocate_margin.md) needs to have clear acceptance criteria defines with acceptance IDs.

### PR DESCRIPTION
**Pull request by cpuja84**
adding ACs to the spec 0011-MARA-check-order-allocate-margin
Closes #826

### CODE REVIEW

File: protocol/0011-MARA-check_order_allocate_margin.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ When an order or other market instruction (amend, cancel, etc.) is submitted we
 
 Orders should be rejected if we can’t allocate sufficient margin.
 
+## Acceptance criteria
+
+1. [ ] Amend an order such that margin requriement is increased and user has enough to top up their margin account so that their order gets submitted successfully. (<a name="0011-MARA-001" href="#0011-MARA-001">0011-MARA-001</a>)
+1. [ ] Amend an order such that margin requriement is increased and user doesn't have enough to top up their margin account so their order gets rejected. (<a name="0011-MARA-002" href="#0011-MARA-002">0011-MARA-002</a>)
+1. [ ] Cancelling an order releases the margin amount back to user's account (<a name="0011-MARA-003" href="#0011-MARA-003">0011-MARA-003</a>)
```
Comment (davidsiska-vega) [inline protocol/0011-MARA-check_order_allocate_margin.md]:
```suggestion
1. [ ] Cancelling an order releases the margin amount back to user's general account, provided the user has no other orders or positions (<a name="0011-MARA-003" href="#0011-MARA-003">0011-MARA-003</a>) 
```

File: protocol/0011-MARA-check_order_allocate_margin.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ When an order or other market instruction (amend, cancel, etc.) is submitted we
 
 Orders should be rejected if we can’t allocate sufficient margin.
 
+## Acceptance criteria
+
+1. [ ] Amend an order such that margin requriement is increased and user has enough to top up their margin account so that their order gets submitted successfully. (<a name="0011-MARA-001" href="#0011-MARA-001">0011-MARA-001</a>)
```
Comment (davidsiska-vega) [inline protocol/0011-MARA-check_order_allocate_margin.md]:
```suggestion
1. [ ] If an order is amended such that margin requirement is increased and user has sufficient balance in the general account to top up their margin account then the amendment is executed successfully. (<a name="0011-MARA-001" href="#0011-MARA-001">0011-MARA-001</a>)
```

File: protocol/0011-MARA-check_order_allocate_margin.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ When an order or other market instruction (amend, cancel, etc.) is submitted we
...
+1. [ ] Amend an order such that margin requriement is increased and user doesn't have enough to top up their margin account so their order gets rejected. (<a name="0011-MARA-002" href="#0011-MARA-002">0011-MARA-002</a>)
```
Comment (davidsiska-vega) [inline protocol/0011-MARA-check_order_allocate_margin.md]:
```suggestion
1. [ ] If an order is amended such that margin requirement is increased and user doesn't have sufficient balance in the general account to top up their margin account then their amend is not executed but the unamended order stays on the book. (<a name="0011-MARA-002" href="#0011-MARA-002">0011-MARA-002</a>)
```

**Review by davidsiska-vega — DISMISSED** [review]
It would be worth speaking with @Jiajia-Cui and either linking to scenarios in feature tests that test the margin allocation for orders or quickly adding a few (use the lognormal model, not the simple risk model ideally in at least one).

**Comment by cpuja84** [comment]
@davidsiska-vega - Please could you approve this PR as I have created a separate [ticket](https://github.com/vegaprotocol/specs-internal/issues/930) for adding lognormal risk acceptance criteria.

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**